common/locator.go

```diff
@@ -23,3 +25,26 @@ func NewLocator(ctx context.Context, selector string, f *Frame, l *Logger) *Loca
 		log:      l,
 	}
 }
+
+// Click on an element using locator's selector with strict mode on.
+func (l *Locator) Click(opts goja.Value) {
+	l.log.Debugf("Locator:Click", "fid:%s furl:%q sel:%q opts:%+v", l.frame.ID(), l.frame.URL(), l.selector, opts)
+
+	var err error
+	defer func() { throwOrSlowMo(l.ctx, err) }()
+
+	copts := NewFrameClickOptions(l.frame.defaultTimeout())
+	if err = copts.Parse(l.ctx, opts); err != nil {
+		return
+	}
+	if err = l.click(copts); err != nil {
+		return
+	}
+}
+
+// click is like Click but takes parsed options and neither throws an
+// error, or applies slow motion.
+func (l *Locator) click(opts *FrameClickOptions) error {
+	opts.Strict = true
```

Comment:
Using `locators` forces the user to work in `strict` mode?

Reply:
Yep, that's the whole idea. Well, mostly.